homeassistant/helpers/httpx_client.py

```diff
@@ -66,6 +66,7 @@ def create_async_httpx_client(
     client = HassHttpXAsyncClient(
         verify=verify_ssl,
         headers={USER_AGENT: SERVER_SOFTWARE},
+        proxies=hass.config.proxies,  # type: ignore[arg-type]
```

Comment:
This seems fairly limited? What about all other HTTP clients that are used in Home Assistant? And what would clients from pulled in Python libraries?